mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/components/SwapSummary.tsx

```diff
@@ -131,14 +131,12 @@ export function SwapSummary({
             <BottomSheetInfoV2
               alertInfo={{
                 title: "Settlements",
-                message: `Settlement block is the pre-determined  block height that this transaction will be executed. 
-                \nSettlement value is based on the oracle price at the settlement block. 
-                \nUsers will be able to cancel this transaction as long as the settlement block has not been executed. 
-              `,
+                message: "",
```

Comment:
Is it intended?

Reply:
Yes, my thought is to keep the current behavior of the bottom modal component so that we can reuse it for future components that does not require the style text as SettlementMessage()

Reply:
Yea, can at least put something in web version, not a priority. We can hide it in web like other info icon or at least paste the text in that prompt and work on it later
![image](https://user-images.githubusercontent.com/13292662/188815842-45b94904-65f3-4743-9b20-ca359e7cccda.png)

Reply:
Hidden in web